frontend/src/types/QuestionTypes.ts

```diff
@@ -1,10 +1,41 @@
+import {
+  QuestionTextMetadata,
+  ShortAnswerMetadata,
+  TextMetadata,
+} from "./QuestionMetadataTypes";
+
+export interface QuestionElement {
+  id: string;
+  type: QuestionElementType;
+  data: QuestionElementDataType;
+  error?: string;
+}
+
 export enum QuestionElementType {
-  QUESTION = "Question",
-  TEXT = "Text",
-  IMAGE = "Image",
-  MULTIPLE_CHOICE = "Multiple Choice",
-  SHORT_ANSWER = "Short Answer",
-  MULTI_SELECT = "Multi-select",
+  QUESTION_TEXT = "QUESTION_TEXT",
+  TEXT = "TEXT",
+  IMAGE = "IMAGE",
+  MULTIPLE_CHOICE = "MULTIPLE_CHOICE",
+  SHORT_ANSWER = "SHORT_ANSWER",
+  MULTI_SELECT = "MULTI_SELECT",
+}
```

Comment:
Updated the naming of this values of this enum to match the corresponding backend enum. Then, in all the places where the value was used (i.e as a text label), used our existing formatting utils.